ncm-metaconfig/src/main/perl/metaconfig.pm

```diff
@@ -44,16 +42,6 @@ sub prepare_action
         }
     }
 
-    if ($srv->{daemon}) {
```

Comment:
I'm not sure I understand...
Ae you saying that you update the component RPMs without updating the schema?